tendrl/monitoring_integration/grafana/grafana_org_utils.py

```diff
@@ -36,6 +36,25 @@ def create_org(org_name):
         raise exceptions.ConnectionFailedException
 
 
+''' Get particular organisation by name '''
+
+def get_org_id(org_name):
+
+    config = maps.NamedDict(NS.config.data)
+    if utils.port_open(config.grafana_port, config.grafana_host):
+        resp = get("http://{}:{}/api/orgs/name/"
+                      "{}".format(config.grafana_host,
+                                  config.grafana_port,
+                                  org_name),
+                   auth=config.credentials)
+        try:
+            return resp._content
```

Comment:
there is no reason to use private variable of a third party library (which can change without notice)

Either use resp.text or resp.content everywhere in monitoring-integration

Reply:
@r0h4n https://pastebin.com/2EAvXShi this is a sample response sent by Grafana.
So to access the content of the response I have to use resp._content

Reply:
You need to study python "properties" and the request library more. using private variables is NOT a good python practice

Use resp.content

More: https://github.com/requests/requests/blob/master/requests/models.py#L811

Reply:
@r0h4n  ack, will read the libraries, thanks
Will update the PR